src/App.css

```diff
@@ -49,9 +43,18 @@ tr {
 
 .footer {
   padding-top: 20px;
+  margin-bottom: 0rem;
   text-align: center;
+  list-style-position: inside;
+  background-color: #0d0980;
+  color: white;
 }
 
 .card {
   width: 18rem;
 }
+.cards-box {
```

Comment:
To think of this as 'mobile first', imagine how it would best look on a small screen first (probably, the 3 cards stacked vertically). Write the styles for that, then add a media query for eg min-width: 768px and adjust the styles inside the media query for the wider screen. If you can get in the habit of writing layout styles mobile-first, it will become simpler to approach it each time, and it will work on all screen sizes.